src/utils/openwhisk.js

```diff
@@ -128,11 +128,6 @@ function extractActionContext(params) {
   // extract content (will be added to context)
   delete disclosed.content;
 
-  // keep logger backward compatible with winston logger
-  if (__ow_logger) {
```

Comment:
I kind of remember this line is mandatory and I am pretty sure that removing it will introduce a regression somewhere (@tripodsan did not add it just for fun ;) ). Did you test this new pipeline with the simulator AND in "production" ?

Based on your comment https://github.com/adobe/helix-cli/issues/1240#issuecomment-564251287, I think that changing the `if` should be enough to fix the issue: 

```
if (__ow_logger && !__ow_logger.log) {
```

to not override an existing `log` function (or something like that ;) )

Reply:
^@kptdobe  yes, hmmm. I searched through Winston's codes history to see when it was ever without a .log function, and didn't find anything where it was different and wouldn't result in the same issue. This might be a powerful lesson in nothing is as it seems :D 

I tested with pipeline installed in helix-cli node_modules during hlx up!

Reply:
the problem is, that winston logger has a `log(level, msg)` method, but helix-log implements `log(msg)` only (following console.log semantics). so while this still "works", it will output a wrong log-message.

but let's keep it this way, as we are going to refactor the logging anyways in the next days.

Reply:
ahh great point, I didn't see this. Thank you.